docs/csharp/deconstruct.md

```diff
@@ -62,7 +62,7 @@ The following example illustrates the use of tuples with discards. The `QueryCit
 
 ### Deconstructing user-defined types
```

Comment:
```suggestion
## Deconstructing user-defined types
```
This should be the parent of the two following sections, which should be changed to have level-3 rather than level-2 heads.